src/geode/model/helpers/component_mesh_queries.cpp

```diff
@@ -137,4 +217,77 @@ namespace geode
             "triangles and quadrangles." );
         return {};
     }
+
+    absl::InlinedVector< PolygonVertices, 2 >
+        block_vertices_from_surface_polygon( const BRep& model,
```

Comment:
We need to discuss what this function does. Looks too much according to its name.

Reply:
and maybe the function is a bit too long